ast/src/parsed/folder.rs

```diff
@@ -25,6 +25,10 @@ pub trait Folder<T> {
                     SymbolValue::Machine(machine) => self.fold_machine(machine).map(From::from),
                     SymbolValue::Import(import) => self.fold_import(import).map(From::from),
                     SymbolValue::Module(module) => self.fold_module(module).map(From::from),
+                    SymbolValue::Expression(e) => {
+                        // Not folding expressions by default
```

Comment:
why?

Reply:
Because the expression folder trait is a different trait. So if you want to change the behaviour on expressions, you have to implement the other trait and then just call that other trait from here.